src/csharp/Grpc.Core/Internal/AsyncCall.cs

```diff
@@ -2,11 +2,11 @@
 
 // Copyright 2015, Google Inc.
 // All rights reserved.
-// 
+//
```

Comment:
Please revert the trailing spaces in the copyright header (3 occurences). The same header is copied all accross the C# code, so if we want to address that, we can do it in a separate PR.

Fine to remove trailing spaces in the rest of the file, as they shouldn't be there.